app/views/templates/_news_and_documents.html.erb

```diff
@@ -26,11 +26,15 @@
         if articles.any? %>
           <articles>
 
-            <% articles.each do |a| %>
+            <% articles.each do |a| %>c
 
               <article-listing category="<%= cms_block_content(:article_type, a) %>" ref="article">
                 <h3><%= link_to a.label, a.url(:relative) %></h3>
-                <%= image_tag cms_block_content(:image, a).file.url(:small), alt: a.label %>
+
+                <% if(!cms_block_content(:image, a).to_s.strip.empty?)
+                  image_tag cms_block_content(:image, a).file.url(:small), alt: a.label
+                end %>
```

Comment:
Try to avoid extending erb tags across lines, each line should be wrapped, especially as we're doing different things here, the `if` on the first line will need a `<%` tag, but we want to output the image, so that'll need a `<%=` tag